cmd/airflow.go

```diff
@@ -162,6 +168,50 @@ func newAirflowInitCmd() *cobra.Command {
 	return cmd
 }
 
+func newAirflowUpgradeTestCmd(astroClient astro.Client) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "upgrade-test",
+		Short:   "Run tests to see if your environment and DAGs are compatable with a new version of Airflow or Runtime",
+		Long:    "Run some tests to see if your environment and DAGs are compatable with a new version of Airflow or Runtime",
```

Comment:
```suggestion
		Long:    "Run tests to see if your environment and DAGs are compatible with a new version of Airflow or Astro Runtime",
```
If you wanted, you could get more specific about what type of tests are run for the long text. ie. Are they pytests, unit tests, do they need Docker, etc